lite/pyproject.toml

```diff
@@ -12,6 +12,9 @@ dependencies = [
     "requests",
     "numpy",
     "shapely",
+    "nltk",
```

Comment:
can we make nltk and rouge_score optional as well?

Reply:
Im just going to omit these for now seeing that the llm metrics aren't added yet